content/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,269 @@
+```
+
+#### 5. In AWS public cluster, verify connectivity has been established.
+
+```markdown
+$ skupper status               
+Skupper is enabled for namespace "aws-boutique" with site name "aws-public" in interior mode. It is connected to 1 other site. It has no exposed services.                                    
+The site console url is:  https://aea867abf6fb6413d8f577652da564c1-130946084.us-east-2.elb.amazonaws.com:8080
+The credentials for internal console-auth mode are held in secret: 'skupper-console-users'
+```
+
+F﻿rom the *Skupper* console URL at **https://aea867abf6fb6413d8f577652da564c1-130946084.us-east-2.elb.amazonaws.com:8080**, you can see the connections from
+ the AWS public cluster and the PCE private cluster:
+
+![](/img/skupper-status.png)
+
+### Expose Application Microservices to Virtual Application Network
+
+#### 1. In PCE private cluster, expose 3 servicess:
+
+```markdown
+$ skupper expose deployment emailservice --address emailservice --port 5000 --protocol http2 --target-port 8080
+deployment emailservice exposed as emailservice
+
+$ skupper expose deployment paymentservice --address paymentservice --port 50051 --protocol http2 --target-port 50051
+deployment paymentservice exposed as paymentservice
+
+$ skupper expose deployment shippingservice --address shippingservice --port 50051 --protocol http2 --target-port 50051
+deployment shippingservice exposed as shippingservice
+```
+
+#### 2. In AWS public cluster, expose 7 services:
+
+```markdown
+$ skupper expose deployment productcatalogservice --address productcatalogservice --port 3550 --protocol http2 --target-port 3550
+deployment productcatalogservice exposed as productcatalogservice
+
+$ skupper expose deployment recommendationservice --address recommendationservice --port 8080 --protocol http2 --target-port 8080
+deployment recommendationservice exposed as recommendationservice
+
+$ skupper expose deployment checkoutservice --address checkoutservice --port 5050 --protocol http2 --target-port 5050
+deployment checkoutservice exposed as checkoutservice
+
+$ skupper expose deployment cartservice --address cartservice --port 7070 --protocol http2 --target-port 7070
+deployment cartservice exposed as cartservice
+
+$ skupper expose deployment currencyservice --address currencyservice --port 7000 --protocol http2 --target-port 7000
+deployment currencyservice exposed as currencyservice
+
+$ skupper expose deployment adservice --address adservice --port 9555 --protocol http2 --target-port 9555
+deployment adservice exposed as adservice                                                      
+
+$ skupper expose deployment redis-cart --address redis-cart --port 6379 --protocol tcp --target-port 6379
+deployment redis-cart exposed as redis-cart                                                    
+```
+
+### Access Online Boutique Application
+
+F﻿rom the *Skupper* console, you can see all he deployed services to the AWS public cluster and the PCE private cluster:  
```

Comment:
..., you can see all the deployed services to the public AWS EKS cluster and the private Kubernetes cluster: